source/_docs/clone-site.md

```diff
@@ -5,8 +5,47 @@ categories: [developing]
 tags: [backups, platform, migrate]
 keywords: clone, restore backup, clone existing pantheon site, clone from pantheon backup, clone pantheon site, copy pantheon site
 ---
-See the following documentation to copy your existing Pantheon site to a second Pantheon site:
 
-For WordPress, see [Migrate to Pantheon: WordPress](/docs/migrate-wordpress/).
+## WordPress
+After selecting **[Migrate Existing Site](https://dashboard.pantheon.io/sites/migrate/)**, you'll create a new Pantheon site and install the Pantheon Migrations plugin on your existing Pantheon site. Visit the plugin settings and paste in your newly created Pantheon site name, as well as the machine token we generate for you, which gives the plugin access to move data to your new Pantheon site.
 
-For Drupal, see [Migrate Sites to Pantheon: Manual Method](/docs/migrate-manual).
+See exactly how it works in the following video:
+<iframe width="420" height="315" src="https://www.youtube.com/embed/3_DjdIueKM4" frameborder="0" allowfullscreen></iframe>
+
+For more information see: [Migrate to Pantheon: WordPress](/docs/migrate-wordpress)
+
+## Drupal
+After selecting **[Migrate Existing Site](https://dashboard.pantheon.io/sites/migrate/)**, you'll create a new Pantheon site. During the **Create Site Archive** step of the migration process, use [Terminus](/docs/terminus) to create an archive of your existing Pantheon site:
+
+```bash
+terminus drush 'ard --destination=sites/default/files/cC1j4cP0GWaMWrD.tgz'
+```
+
+This operation writes the archive to Pantheon's filesystem in a web accessible location (e.g. `http://env-site-name.pantheonsite.io/sites/default/files/cC1j4cP0GWaMWrD.tgz`). Click **Continue Migration** and follow all remaining instructions within the guided migration process.

```

Comment:
Did you successfully clone Dev and a non-dev environment? Just wondering if we should recommend they clone from Dev?

Would like to include a placeholder for the filename here and below, something like ` --destination=sites/default/files/<RANDOM_HASH>.tgz` ?